packages/ag-charts-community/src/api/agCharts.ts

```diff
@@ -263,7 +263,7 @@ class AgChartsInternal {
                     }
                 }
             },
-            { skip: ['data'] }
+            new Set('data')
```

Comment:
```suggestion
            new Set(['data'])
```
Missing array wrap, meaning the set is equal to `{ 'd', 'a', 't' }`

Reply:
Thanks, good spot